packages/forms-web-app/src/controllers/examination/your-interested-party-number.js

```diff
@@ -0,0 +1,71 @@
+const {
+	routesConfig: {
+		examination: {
+			pages: {
+				yourInterestedPartyNumber: {
+					view: yourInterestedPartyNumberView,
+					route: yourInterestedPartyNumberRoute
+				},
+				haveAnInterestedPartyNumber: { route: haveAnInterestedPartyNumberRoute },
+				submittingFor: { route: submittingForRoute }
+			}
+		}
+	},
+	routesConfig: { examination }
```

Comment:
As this is only being used to extract the directory would it be better to add it to line 4 and declare it as 'examinationDirectory'? `directory: { examinationDirectory }` This would also be in keeping with the other controllers.

Reply:
it was not being used so I had to remove it for lint purposes